shell/vue.config.js

```diff
@@ -498,6 +499,8 @@ module.exports = function(dir, _appConfig) {
     },
     outputDir,
     pages: {
+      // Taken from @nuxt/vue-app/template/index.js
```

Comment:
not sure this comment is in the right place

Reply:
This is where we define the entry, so it's worth adding it here, it doesn't cause any harm to have some extra information.